Misc/NEWS.d/3.14.0b4.rst

```diff
@@ -75,7 +75,7 @@ to the HTML5 standard.
 * Multiple ``=`` between attribute name and value are no longer collapsed.
   E.g. ``<a foo==bar>`` produces attribute "foo" with value "=bar".
 
-* Whitespaces between the ``=`` separator and attribute name or value are no
+* [REVERTED] Whitespaces between the ``=`` separator and attribute name or value are no
```

Comment:
Yes, this is fine. Maybe we can just give a hint as to when, like:

```suggestion
* [Reverted in :gh:`136927`] Whitespaces between the ``=`` separator and attribute name or value are no
```

That way people can easily find out why.